contactme.css

```diff
@@ -89,43 +85,41 @@
   border: #fff 1px solid;
 }
 
-.imgcontainer-contact{
- margin-top: 3.5rem;
- display: flex;
- justify-content: center;
+.imgcontainer-contact {
+  margin-top: 3.5rem;
+  display: flex;
+  justify-content: center;
 }
 
 @media only screen and (min-width: 720px) {
```

Comment:
- [ ] The value for min-width property should be in range of 992px - 1100px. These are the accepted desktop screen breakpoints. Change your value. What you are using lies under the tablet screen breakpoints.